package.json

```diff
@@ -32,9 +32,10 @@
   },
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
+    "@edx/browserslist-config": "^1.5.0",
```

Comment:
`browserslist-config` is required for webpack to build the MFE so it should be a dependency, not a devDependency.

Ref: [OEP-67 - Frontend Technology Selection - 15 Development dependencies should be separated into devDependencies](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0067-bp-tools-and-technology.html#frontend-technology-selection)

Reply:
If it doesn't need to be in the bundle that is served to users it should not be a direct dependency. We've discussed this in frontend working group a few times and it seems we need to update the OEP. Adding it as a direct dependency increases the bundle size which is not something we want.